### PR TITLE
shields: enable SD card support on Adafruit 2.8 TFT Touch v2 and Waveshare Epaper

### PR DESCRIPTION
**Pull request by jfischer-no**
Enable SD card support on Adafruit 2.8 TFT Touch v2 and Waveshare Epaper shields.

### CODE REVIEW

File: boards/shields/waveshare_epaper/waveshare_epaper.dtsi [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```
Comment (erwango) [inline boards/shields/waveshare_epaper/waveshare_epaper.dtsi:2]: Are you ? :-)

File: boards/shields/waveshare_epaper/waveshare_epaper.dtsi [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		   <&arduino_header 12 GPIO_ACTIVE_LOW>;	/* D04 */
+
+	sdhc0: sdhc@1 {
+		compatible = "zephyr,mmc-spi-slot";
+		reg = <1>;
+		status = "okay";
+		label = "SDHC0";
+		spi-max-frequency = <24000000>;
+	};
+};
```
Comment (erwango) [inline boards/shields/waveshare_epaper/waveshare_epaper.dtsi:19]: For clarity this file would better be renamed as `waveshare_epaper_common.dtsi` or `waveshare_epaper_sdhc.dtsi`

**Review by erwango — CHANGES_REQUESTED** [review]
Minor comments

File: boards/shields/waveshare_epaper/waveshare_epaper.dtsi (hunk 1 above)
Comment (jfischer-no) [reply boards/shields/waveshare_epaper/waveshare_epaper.dtsi:2]: yes :-)

File: boards/shields/waveshare_epaper/waveshare_epaper.dtsi (hunk 2 above)
Comment (jfischer-no) [reply boards/shields/waveshare_epaper/waveshare_epaper.dtsi:19]: I will rename it to waveshare_epaper_common.dtsi

File: boards/shields/waveshare_epaper/waveshare_epaper.dtsi (hunk 1 above)
Comment (erwango) [reply boards/shields/waveshare_epaper/waveshare_epaper.dtsi:2]: Haaaa and that's explain the "-no"

File: boards/shields/waveshare_epaper/waveshare_epaper.dtsi (hunk 2 above)
Comment (erwango) [reply boards/shields/waveshare_epaper/waveshare_epaper.dtsi:19]: That would be fine.

**Review by erwango — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Pull request merged**